department-of-veterans-affairs/abd-vro
# Add application-end2end-test.yml

## What was the problem?


Follow-up to #807 

## How does this fix it?


Reduce maintenance of `end2end-test.yml` GH Action by using the new `application-end2end-test.yml` and `scripts/setenv.sh`.
- remove SLACK_EXCEPTION_CHANNEL env var
- remove SWAGGER_ENABLED

Refactoring:
- make `application-k8s.yml` and `application-local.yml` comparable (by having the same setting names)
- make `application-nonprod.yml` and `application-prod.yml` comparable (by having the same setting names)

## How to test this PR
Ensure VRO runs locally and in end2end GH Action.
Deploy to LHDI and test VRO APIs.
